.github/workflows/build_documentation.yml

```
@@ -69,9 +69,8 @@ jobs:
 
       - name: Make documentation
         run: |
-          version=$(pip show optimum | grep Version | cut -c 10-20) &&
           cd doc-builder &&
-          doc-builder build optimum ../optimum/docs/source --build_dir ../doc-build --clean --version $version --html &&
+          doc-builder build optimum ../optimum/docs/source --build_dir ../doc-build --clean --version main --html &&
```

```suggestion
          doc-builder build optimum ../optimum/docs/source --build_dir ../doc-build --clean --html &&
```

whether it is is `master` or `main` branch gets detected automatically 
see [here](https://github.com/huggingface/accelerate/blob/main/.github/workflows/build_documentation.yml#L70-L73)

---

@lewtun lets merge it & try if the doc build in correct place? 😊

---

companion [moon-landing pr](https://github.com/huggingface/moon-landing/pull/2329)

---

Actually, for `optimum` we have to specify the branch manually for now because the repo has a different structure than the other HF libraries (no `__init__.py` in the root of the repo where we can store the `__version__` attribute). Without it, one gets the following error:

```python
AttributeError: module 'optimum' has no attribute '__version__'
```

I plan to fix this next week, so let's keep the manual flag for now. Gently pinging @echarlaix for permission to merge this :)

---

oh I see, then in that case ignore my suggested change
the hardcoded  `--version main` works too